data/sql/updates/pending_db_world/rev_1731309887413406963.sql

```diff
@@ -0,0 +1,6 @@
+--
+DELETE FROM `smart_scripts` WHERE `entryorguid` = 18631 AND `id` = 1;
+
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `event_param6`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`)
+VALUES
+(18631, 0, 1, 0, 13, 0, 100, 0, 5000, 10000, 0, 0, 0, 0, 11, 15614, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Cabal Cultist - On Victim Casting - Cast \'Kick\' with 5-10 second cooldown');
```

Comment:
If you don't set a repeat timer, they will be cast only once.

Reply:
you're using SAI event 13 VICTIM_CASTING, which only has one pair of timers - repeatMin and repeatMax. @Nyeriah probably referred to event 0 UPDATE_OC which has two pair of timers - initialMin/initialMax and repeatmin/repeatMax

Reply:
Okay, I'll fix it next week, I'm on vacation.